TESTS.md

```diff
@@ -27,4 +27,20 @@ Set the `id` section to the id of an existing Google Sheet.
 2. Wait at least the configured amount of milliseconds as configured under `interval` in the configuration file (default 5000).
 
 ### Postconditions
-- The changes are correctly converted and written back to the resource `http://localhost:3000/example/software`.
\ No newline at end of file
+- The resource `http://localhost:3000/example/software` contains the new data.
+
+##  Test if changes on the Pod are synced back to the Google Sheet
+
+### Preconditions
+- Follow and execute all steps in the "cold start" test above.
+
+### Steps
+The pod can be updated with the following request:
+```curl
```

Comment:
Does markdown support `curl`? If not then I would expect `shell`.

Reply:
It does, but shell seems to be better. I'll change it.